tests/lcs/core/image/bug-19668.livecodescript

```diff
@@ -0,0 +1,58 @@
+script "CoreEngineClipboard"
```

Comment:
I suggest these tests should go in 'lcs/core/interface/image-undo.livecodescript' with name CoreInterfaceImageUndo.